### PR TITLE
Fix v2 site issues

### LINKED ISSUE #646: Update site components and markup to use USWDS 2.0
- [x] hero
- [x] footer
- [x] grid
- [x] sidenav
- [x] mobile navigation
  - mobile menu button on the left instead of right side 
- [x] banner hide/show text disappears

### PR DESCRIPTION
**Pull request by maya**
Fixes the sidenav, mobile nav, and several other issues.

There were some issues where `usa-prose` style was leaking into components: lists leaking into accordion.

Fixes #646. 

~~Issue with the banner text: hide/show.~~

[😎 Preview](https://federalist-proxy.app.cloud.gov/preview/uswds/uswds-site/fix-site-quirks)

### CODE REVIEW

**Comment by maya** [comment]
One quirk I discovered was the 2nd nested grandchild (3rd level) did not get an indentation:
screen shot 2018-09-25 at 12 46 33 pm

Everything below Color should be indented to show the proper hierarchy.

cc @thisisdano

**Comment by maya** [comment]
@thisisdano is it intentional that the sidenav children have the same font size as the parent? In the navigation it's smaller. In our components, they are both small:

## Current
screen shot 2018-09-25 at 1 28 51 pm

## Mobile nav (smaller children)
screen shot 2018-09-25 at 1 30 11 pm

**Comment by thisisdano** [comment]
I'll check on the indentation and size things. The sizing is not intentional — something must have gotten screwy in the last change. And the indent looks ok on the Fractal side, I'll check the markup here...

**Comment by maya** [comment]
>And the indent looks ok on the Fractal side, I'll check the markup here...
Where are you seeing this? I'm not seeing a 3rd level nav item in the mobile header in fractal. Only the sidenav.

**Comment by thisisdano** [comment]
Oh, that's what I meant, in the sidenav — I hadn't checked it with the mobile nav. Misunderstood what you meant.

**Comment by thisisdano** [comment]
Come to think of it, I'm not really sure how grandchildren display in the nav at all...

**Comment by maya** [comment]
Yes, the screenshot above is of the mobile nav. The sidenav indentation is fine:
screen shot 2018-09-25 at 2 06 01 pm

**Comment by thisisdano** [comment]
Lemme try and duplicate this on the Fractal side so I can fix the source code.

**Comment by maya** [comment]
>Come to think of it, I'm not really sure how grandchildren display in the nav at all...

Before your most recent nav PR, grandchildren in the mobile nav had the same indentation as the sidenav. As they were using the same source.

**Comment by maya** [comment]
I'm seeing some indentation, but not sure if it's correct. See the following pages:

https://federalist-proxy.app.cloud.gov/preview/uswds/uswds-site/fix-site-quirks/style/color/theme-palette/
screen shot 2018-09-25 at 2 29 36 pm
Underneath Color, we want Overview and Theme palette indented. The children of Theme palette are indented, which is good.

See: https://federalist-proxy.app.cloud.gov/preview/uswds/uswds-site/fix-site-quirks/utilities/color/

**Comment by thisisdano** [comment]
So this has _great-grandchildren_...

**Comment by thisisdano** [comment]
Just make sure each child/grandchild/etc list is wrapped in a `ul.usa-nav-submenu` and I'll add great-grandchild spacing to the source styles.

**Comment by maya** [comment]
Will check, the way these menus are constructed in jekyll are really complicated.

**Comment by thisisdano** [comment]
Added great-grandchild indenting in https://github.com/uswds/uswds/pull/2779

**Comment by maya** [comment]
@thisisdano Made sure the correct class was added here aa0e77e.

File: css/_project-custom.scss [hunk 1]
```diff
@@ -144,50 +150,14 @@ body {
...
-      .usa-sidenav-sub_list {
-        a {
-          padding-left: 4.8rem;
-        }
-      }
-    }
+  .usa-nav-sublist {
+    @include margin(0 null);
+    list-style-type: none;
+    padding-left: 0;
```
Comment (maya) [inline css/_project-custom.scss]: Had to remove some list styles bc of list styles, which wasn't great.

**Comment by thisisdano** [comment]
Use `usa-nav-submenu` for the submenus in the nav, not `usa-nav-sublist` — it _should_ work when that changes:

screen shot 2018-09-25 at 3 12 02 pm

**Comment by maya** [comment]
Updated in ae77521, should be good for review.

**Comment by maya** [comment]
Possibly, this situation is bc of it's following a grid item not text. I'd relegate this to another PR / issue tho.

**Comment by maya** [comment]
Previously we said folks would be expected to add custom margins when they encountered something like this.

**Comment by thisisdano** [comment]
Yeah, exactly — not for this PR — just wanted to bring it up

File: css/styleguide.scss [hunk 2]
```diff
@@ -5,4 +5,4 @@
 @import 'uswds/dist/scss/packages/uswds-layout-grid';
 @import 'uswds/dist/scss/packages/uswds-components';
 @import 'project-custom';
-@import 'uswds/dist/scss/packages/uswds-utilities';
\ No newline at end of file
+@import 'uswds/dist/scss/packages/uswds-utilities';
```
Comment (thisisdano) [inline css/styleguide.scss]: since we're adding this, should we just `@import uswds` before the project custom?

Comment (maya) [reply css/styleguide.scss]: Not sure I follow, this showed up as green bc I added an extra line at the end of the file, but didn't change anything. We want utilities to override custom code.

Comment (thisisdano) [reply css/styleguide.scss]: oh i see — utilities may still not override custom code, but ok as is

**Comment by maya** [comment]
That's odd, Circle and Federalist are linking to the branch `dw-remove-bourbon`, which is not this PR's branch. Those tests already passed. OK to merge?

**Pull request merged**